ebos/equil/initstateequil_impl.hh

```diff
@@ -1669,17 +1676,12 @@ applyNumericalAquifers_(const GridView& gridView,
         const auto search = num_aqu_cells.find(cartIx);
         if (search != num_aqu_cells.end()) {
             // numerical aquifer cells are filled with water initially
-            // for co2store the oilphase may be used for the brine
-            bool co2store_oil_as_brine = co2store && FluidSystem::phaseIsActive(FluidSystem::oilPhaseIdx);
-            const auto watPos =  co2store_oil_as_brine? FluidSystem::oilPhaseIdx : FluidSystem::waterPhaseIdx;
             if (FluidSystem::phaseIsActive(watPos)) {
                 this->sat_[watPos][elemIdx] = 1.;
-            } else {
-                throw std::logic_error  { "Water phase has to be active for numerical aquifer case" };
             }
```

Comment:
Technically `if (FluidSysten::phaseIsActive(watPos))` is redundant with the check already outside the loop.  You could just replace this conditional assignment with an unconditional one,
```
this->sat_[watPos][elemIdx] = 1.0;
```
instead.

Reply:
Indeed, thanks! I also put `const auto oilPos = FluidSystem::oilPhaseIdx;` and `const auto gasPos = FluidSystem::gasPhaseIdx;` outside the loop since they are not element dependent.